tests/darwin/dataset/remote_dataset_test.py

```diff
@@ -63,6 +64,237 @@ def create_annotation_file(
         json.dump(annotation_content, f)
 
 
+@pytest.fixture()
+def files_content() -> dict:
+    return {
+        "items": [
```

Comment:
We seem to be leaking some valid signed URLs here, are we ok with it?

On a separate note, should we keep these large payloads in actual json files, so we only open and decode them here? That would make this code cleaner, and a centralized place for example payloads to reuse in tests might be beneficial.

Reply:
This file is the decoded (into Python format) json, not a _real_ json, file. 
We could have a folder for that, I am OK with either approach.